libs/designsystem/src/lib/components/chart/configs/type.config.ts

```diff
@@ -117,27 +68,37 @@ export const CHART_TYPE_CONFIGS = {
       },
       elements: {
         line: {
-          spanGaps: true,
           tension: 0.2,
           borderColor: getThemeColorHexString('medium'),
         },
         point: {
-          pointRadius: 0,
+          radius: 0,
+        },
+      },
+    },
+  },
+  line: {
+    type: 'line',
+    options: {
+      elements: {
+        point: {
+          radius: 0,
         },
       },
     },
   },
 };
 
+/* TODO: Find a proper place to store these two */
```

Comment:
Remember todo :)

Reply:
I've found a proper place to store these two! 
Todo resolved :D